app/serializers/event_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class EventSerializer < ActiveModel::Serializer
```

Comment:
Also another downside is the fields need to match the active record model, meaning I can't convert the keys to camelCase.  Summary is: really simple, not much code, but not much customization allowed either.